tools/upload-gh-pages.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Upload website to gh-pages
+USAGE="$0 <html_dir> <project-name> [<organization-name>]"
+HTML_DIR=$1
+if [ -z "$HTML_DIR" ]; then
+    echo $USAGE
+    exit 1
+fi
+PROJECT=$2
+if [ -z "$PROJECT" ]; then
+    echo $USAGE
+    exit 1
+fi
+ORGANIZATION=$3
+if [ -z "$ORGANIZATION" ]; then
+    ORGANIZATION=nipy
+fi
+upstream_repo="https://github.com/$ORGANIZATION/$PROJECT"
+cd $HTML_DIR
```

Comment:
Can you make sure this command succeeds, or that the `$HTML_DIR` exists? This is the major source of user error for me... and though this command fails, something gets uploaded (but the wrong thing).